guessing_game/src/main.rs

```diff
@@ -0,0 +1,41 @@
+use std::io;
+use rand::Rng;
+use std::cmp::Ordering;
+
+fn main() {
+    let secret_number = rand::thread_rng().gen_range(1..=100);
+
+    println!("{secret_number}");
+
+    println!("Hello, and welcome to the guessing game!");
+
+    let mut guess;
+
+    loop {
+        println!("Please enter your input:");
+
+        guess = String::new();
+
+        io::stdin()
+            .read_line(&mut guess)
+            .expect("Failed to read line!");
```

Comment:
bishbash

Reply:
i have changed this in a new commit please take a look